app/views/foods/_food.html.erb

```diff
@@ -1,27 +1,38 @@
-<div id="<%= dom_id food %>">
-  <p>
-    <strong>Name:</strong>
-    <%= food.name %>
-  </p>
+<%# <table id="foods-table">
+  <tr>
+    <th>
+    Food
+    </th>
 
-  <p>
-    <strong>Measurementunit:</strong>
-    <%= food.measurementUnit %>
-  </p>
+    <th>
+    Measurement unit
+    </th>
+    <th>
+    Unit price
+    </th>
+     <th>
+    Actions
+    </th>
+    
+   
+  </tr>
+
+  <tr>
+    <td>
+     <%= food.name %>
+    </td>
 
-  <p>
-    <strong>Price:</strong>
+    <td>
+    <%= food.measurementUnit %>
+    </td>
+    <td>
     <%= food.price %>
-  </p>
+    </td>
+     <td>
+    Delete
+    </td>
+  <%# </tr> %>
```

Comment:
- [ ] I logged out of the initial account and created a new one so I could verify if a user doesn't have access to the delete button if they aren't the creator or owner of a food item but this wasn't the case. I was still able to see the delete button for the food items the previously logged-in user created. So kindly fix this as well :+1: 
- [ OPTIONAL ] It would be great if you added a `Logout button` to your homepage. I had to create my own :+1:  
- When logged in as `Test` user with `ID 2`
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/d1dc5504-f721-4557-baf0-afa7f3584ce8)

- [ ] The same issue persists on the `Recipe Index page` as well. I was able to see and delete the recipe created by the previous user so kindly fix this as well.  You should only be able to remove a recipe item only if you're the owner :+1: 
- When logged in as `Test` user with `ID 2`
![image](https://github.com/ridabensalem/recipe-app/assets/102699958/4e2332ca-0b71-47bb-bdb3-3d6022c402b2)

Reply:
Fixed.